### PR TITLE
Add payload example for os installlation workflow

### PR DESCRIPTION
**Pull request by cgx027**
Two payload files for each OS, one with all parameters included, the
other one with the minimum required parameters.

### CODE REVIEW

File: example/samples/install_esx_payload_full.json [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+                    },
+                    "ipv6": {
+                        "ipAddr": "fec0::6ab4:0:5efe:159.45.14.11",
+                        "gateway": "fe80::5efe:131.107.25.100",
+                        "netmask": "ffff.ffff.ffff.ffff.0.0.0.0",
+                        "vlanIds": [106, 108]
+                    }
+                }
+            ],
+            "kvm": true,
```
Comment (yyscamper) [inline example/samples/install_esx_payload_full.json]: Esxi doesn't support kvm

File: example/samples/install_esx_payload_full.json [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+            "version": "6.0",
+            "repo": "http://172.31.128.1:9080/esxi/6.0",
+            "rootPassword": "RackHDRocks!",
+            "hostname": "rackhd-node",
+            "domain": "abc.com",
+            "users": [
+                {
+                    "name": "rackhd1",
+                    "password": "123456",
+                    "uid": 1010,
```
Comment (yyscamper) [inline example/samples/install_esx_payload_full.json]: Esxi doesn't support `uid`, so I suggest remove this line.

**Comment by yyscamper** [comment]
:+1: 

@RackHD/corecommitters 
@WangWinson  for Ubuntu payload
@zyoung51 & @pengz1 for ESXi payload

**Comment by WangWinson** [comment]
:+1:

**Comment by anhou** [comment]
:+1:

File: example/samples/install_esx_payload_full.json [hunk 3]
```diff
@@ -0,0 +1,75 @@
+{
+    "options": {
+        "defaults": {
+            "version": "6.0",
+            "repo": "http://172.31.128.1:9080/esxi/6.0",
+            "rootPassword": "RackHDRocks!",
+            "hostname": "rackhd-node",
+            "domain": "abc.com",
```
Comment (davidzuhn) [inline example/samples/install_esx_payload_full.json]: examples should use example.com or another domain reserved for examples in RFC 6761.  Don't use a domain that's in existence and is unrelated to RackHD.

Comment (benbp) [inline example/samples/install_esx_payload_full.json]: agreed

**Comment by benbp** [comment]
Fixes https://github.com/RackHD/RackHD/issues/159

File: example/samples/install_esx_payload_full.json (hunk 3 above)
Comment (cgx027) [inline example/samples/install_esx_payload_full.json]: Good comment @davidzuhn , let me fix this.

**Pull request merged**